src/main/java/jenkins/plugins/svnmerge/FeatureBranchProperty.java

```diff
@@ -75,18 +75,27 @@
      */
     private String upstream;
     private transient RebaseAction rebaseAction;
+    
+    private final String commitPrefix;
 
     @DataBoundConstructor
-    public FeatureBranchProperty(String upstream) {
+    public FeatureBranchProperty(String upstream, String rebaseCommitPrefix) {
         if (upstream == null) {
             throw new NullPointerException("upstream");
         }
         this.upstream = upstream;
+        this.commitPrefix = (rebaseCommitPrefix == null || "".equals(rebaseCommitPrefix)) ? 
+				RebaseAction.COMMIT_MESSAGE_PREFIX : rebaseCommitPrefix;;
```

Comment:
I will do the change to compute the prefix during the rebase operation. Also, I agree that's probably better to append a space between the prefix and the commit message.